### PR TITLE
added subtraction function

### PR DESCRIPTION
**Pull request by hazratali-uydevelopers**
added subtraction function, accept request

### CODE REVIEW

File: practice.cpp [hunk 1]
```diff
@@ -5,7 +5,12 @@ int add(int a, int b) {
     return a + b;
 }
 
+int sub(int a, int b) {
```
Comment (Yousuf-Gazi) [inline practice.cpp:8]: what changes you made?

Comment (hazratali-uydevelopers) [reply practice.cpp:8]: i added a function to do subtraction

Comment (Yousuf-Gazi) [reply practice.cpp:8]: great

**Review by Yousuf-Gazi — APPROVED** [review]
good implementation.

**Pull request merged**